src/System.Net.Http/tests/FunctionalTests/PostScenarioTest.cs

```diff
@@ -173,7 +173,23 @@ public PostScenarioTest(ITestOutputHelper output)
             var credential = new NetworkCredential(UserName, Password);
             await PostUsingAuthHelper(serverUri, ExpectedContent, content, credential, preAuthenticate: true);
         }
-        
+
+        [OuterLoop] // TODO: Issue #11345
+        [Theory, MemberData(nameof(EchoServers))]
+        public async Task PostNoContent_ExcludeContentTypeHeader_Success(Uri serverUri)
```

Comment:
The test should be renamed.  "ExcludeContentTypeHeader" is not the condition being tested but rather the expected result.

I would suggest the test be named something like "PostAsync_EmptyContent_ContentTypeHeaderNotSent"